code-of-conduct.md

```diff
@@ -1 +1,76 @@
 # Triangle DevOps Code of Conduct
+
+Triangle DevOps is dedicated to providing a harassment-free community experience
+for everyone, regardless of gender, gender identity and expression, sexual
+orientation, disability, physical appearance, body size, race, age or religion.
+We do not tolerate harassment of community participants in any form.
+
+Community participants violating these rules may be sanctioned or expelled from
+the community at the discretion of the organizers.
+
+Harassment includes, but is not limited to:
+
+* Verbal comments that reinforce social structures of domination related to
+  gender, gender identity and expression, sexual orientation, disability,
+  physical appearance, body size, race, age, or religion.
+* Sexual images in public spaces
+* Deliberate intimidation, stalking, or following
+* Harassing photography or recording
+* Sustained disruption of talks or other events
+* Inappropriate physical contact
+* Unwelcome sexual attention
+* Advocating for, or encouraging, any of the above behaviour
+
+If you are being harassed, notice someone else is being harassed, or have other
+concerns, please contact an organizer immediately.
+
+## Enforcement
+
+Participants asked to stop any harassing behavior are expected to comply
+immediately. If a participant engages in harassing behaviour, the organisers
+retain the right to take any actions to keep events and online spaces as
+welcoming environments for all participants.
+
+This includes warning the offender or expulsion from meetings, events, and any
+online spaces this group uses presently or decides to use in the future.
+
+These rules apply at all meetings, events, meeting or event-related social
+activities (e.g. a meal or beer before or after the main event). These rules
+also apply to the group's online spaces.
+
+## Reporting
+
+If someone makes you or anyone else feel unsafe or unwelcome, please report it
+as soon as possible. Harassment and other code of conduct violations reduce the
+value of our events and online spaces for everyone. We want you to be happy at
+our meetings and events and participating in our online spaces. People like you
+make them all better.
+
+* Talking to the on-site organizers or organizers, who will introduce themselves
+  at the beginning any event
+* Emailing the organizing team [FIXME: email address for contact]
+
+When taking a report in person, our staff will ensure you are safe and cannot be
+overheard. You won't be asked to confront anyone and we won't tell
+anyone who you are.
```

Comment:
A question on this: can we promise completely anonymity? I'm thinking in this case of Chef's internal harassment policy, which contains this note:

"Investigations will remain as confidential as they can, but complete confidentially cannot be guaranteed. Due to the nature of investigations, some information may need to be shared but only on a need to know basis."